### PR TITLE
fix logo not rendered on light theme and update toolkit IAM login form styling

### PR DESCRIPTION
**Pull request by Will-ShaoHua**
## Toolkit

main-page-light

main-page-dark

sso-form-light

sso-form-dark

iam-form-light

iam-form-dark

sso-ing-light

sso-ing-dark

iam-ing-light

iam-ing-dark

## Q

q-main-dark

q-main-light

q-sso-dark

q-sso-light

q-ing-dark

q-ing-lgith

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

## Checklist
- [ ] My code follows the code style of this project
- [ ] I have added tests to cover my changes
- [ ] A short description of the change has been added to the **[CHANGELOG](https://github.com/aws/aws-toolkit-jetbrains/blob/master/CONTRIBUTING.md#contributing-via-pull-requests)** if the change is customer-facing in the IDE.
- [ ] I have added metrics for my changes (if required)
 
## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by andrewyuq — APPROVED** [review]

File: plugins/core/webview/src/q-ui/components/selectableItem.vue [hunk 1]
```diff
@@ -20,7 +20,7 @@
                 />
             </svg>
             <svg
-                v-if="itemTitle == 'Workforce'"
+                v-if="itemId === LoginOption.ENTERPRISE_SSO"
```
Comment (andrewyuq) [inline plugins/core/webview/src/q-ui/components/selectableItem.vue:23]: Enterprise string is differnt for Toolkit panel and Q panel, did we handle this?

Comment (Will-ShaoHua) [reply plugins/core/webview/src/q-ui/components/selectableItem.vue:23]: not yet

Comment (Will-ShaoHua) [reply plugins/core/webview/src/q-ui/components/selectableItem.vue:23]: so we say `workforce` in Toolkit and `professional license` in Q ?

Comment (Will-ShaoHua) [reply plugins/core/webview/src/q-ui/components/selectableItem.vue:23]: nm let me ask in our chat and take it as a followup

Comment (andrewyuq) [reply plugins/core/webview/src/q-ui/components/selectableItem.vue:23]: >so we say workforce in Toolkit and professional license in Q ?
not sure for this part

**Review by rli — APPROVED** [review]

**Pull request merged**